HW_3/app.py

```diff
@@ -0,0 +1,46 @@
+from flask import Flask, render_template, redirect, url_for, flash
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, SubmitField
+from wtforms.validators import InputRequired, Email, Length
+from flask_sqlalchemy import SQLAlchemy
+from werkzeug.security import generate_password_hash, check_password_hash
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'HomeWork3'
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///users.db'  
+db = SQLAlchemy(app)
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String(50), nullable=False)
+    last_name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(200), nullable=False)
+
+class RegistrationForm(FlaskForm):
+    first_name = StringField('Имя', validators=[InputRequired(), Length(min=2, max=50)])
+    last_name = StringField('Фамилия', validators=[InputRequired(), Length(min=2, max=50)])
+    email = StringField('Email', validators=[InputRequired(), Email(), Length(max=120)])
```

Comment:
Тут лучше использовать `EmailField` иначе не будет проверяться , что в поле действительно введен `email`.
```suggestion
    email = EmailField('Email', validators=[InputRequired(), Email(), Length(max=120)])
```